### PR TITLE
[DO NOT MERGE] Style checker pre-cleaner

### PR DESCRIPTION
**Pull request by shosti**

### CODE REVIEW

File: exe/circlemator [hunk 1]
```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
+lib = File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'optparse'
-require_relative '../lib/circlemator/build_canceler'
-require_relative '../lib/circlemator/self_merger'
-require_relative '../lib/circlemator/github_repo'
-require_relative '../lib/circlemator/style_checker'
+require 'circlemator/build_canceler'
```
Comment (shosti) [inline exe/circlemator]: This is a comment that shouldn't be deleted.

**Comment by ukd1** [comment]
@shosti what is this?

**Comment by ukd1** [comment]
@shosti can you close or finish this please

**Comment by shosti** [comment]
Experiment that didn't quite pan out. Closing for now.

**Pull request closed without merging**